lib/v2/storyfm/episodes.js

```diff
@@ -0,0 +1,65 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://storyfm.cn';
+    const currentUrl = `${rootUrl}/episodes-list`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    let items = $('.e-ep')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const a = item.find('h2.e-ep__title a');
+
+            return {
+                title: a.text(),
+                link: a.attr('href'),
+                pubDate: parseDate(item.find('.whitespace-nowrap').text()),
+                itunes_item_image: item.find('.zoom-image-container-progression img').attr('src'),
+            };
```

Comment:
```suggestion
                itunes_item_image: item.find('.zoom-image-container-progression img').attr('src'),
                enclosure_type: 'audio/mpeg',
                enclosure_url: item.find('audio source').attr('src'),
            };
```
Audio URL is also available in `currentUrl`